planner/core/find_best_task.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/pingcap/tidb/sessionctx/stmtctx"
 	"github.com/pingcap/tidb/statistics"
 	"github.com/pingcap/tidb/types"
+	u "github.com/pingcap/tidb/util"
```

Comment:
maybe rename it to `tidbutil` is better just like  
https://github.com/pingcap/tidb/blob/8f81ec150d1c9898bd06a20ad2b52f028066c989/ddl/ddl_worker.go#L35

Reply:
Yes, I have changed it.